src/main/java/no/unit/nva/testutils/HandlerRequestBuilder.java

```diff
@@ -76,4 +77,28 @@ public static String toString(InputStream inputStream) {
             .lines()
             .collect(Collectors.joining(DELIMITER));
     }
+
+    public T getBody(TypeReference<T> typeRef) throws JsonProcessingException {
```

Comment:
Necessary for being able to update a request through different steps in cucumber